packages/lodestar/src/network/peers/peerManager.ts

```diff
@@ -215,6 +215,19 @@ export class PeerManager {
     this.pingAndStatusTimeouts();
   }
 
+  hasBeaconCommitteeSubnetPeer(subnet: number): boolean {
```

Comment:
This function should look directly into gossipsub data, not the peer's metadata. Peers can lie, so what's the ultimate truth is the gossipsub state. @wemeetagain what do you recommend to look at?